Week2/arrays.js

```diff
@@ -0,0 +1,12 @@
+console.log(` 
+2. Arrays!`); // how to add or remove input in an array.
+let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.shift();
+favoriteAnimals.unshift('blowfish', 'meerkat');

```

Comment:
This works, but can you find a way to use the `splice` function?

Reply:
Yes
instead of Line 6 we can use
favoriteAnimals.splice(0, 1);